parquet-avro/README.md

```diff
@@ -44,3 +44,80 @@ Apache Avro integration
 | `parquet.avro.add-list-element-records` | `boolean` | Flag whether to assume that any repeated element in the schema is a list element.<br/>The default value is `true`. |
 | `parquet.avro.write-parquet-uuid`       | `boolean` | Flag whether to write the [Parquet UUID logical type](https://github.com/apache/parquet-format/blob/master/LogicalTypes.md#uuid) in case of an [Avro UUID type](https://avro.apache.org/docs/current/spec.html#UUID) is present.<br/>The default value is `false`. |
 | `parquet.avro.writeFixedAsInt96`    | `String` | Comma separated list of paths pointing to Avro schema elements which are to be converted to `INT96` Parquet types.<br/>The path is a `'.'` separated list of field names and does not contain the name of the schema nor the namespace. The type of the referenced schema elements must be `fixed` with the size of 12 bytes.<br/>**NOTE: The `INT96` Parquet type is deprecated. This option is only to support old data.** |
+
+## Apache Parquet-Avro is a combination of two technologies:
+
+1. **Apache Parquet**: A columnar storage format for Hadoop and other big data platforms, optimized for querying large datasets.
+
+2. **Apache Avro**: A data serialization system that provides a compact, fast, and efficient way to serialize and deserialize data.
+
+## Major aspects of Apache Parquet-Avro:
```

Comment:
IMHO, we don't need these kinds of descriptions in the parquet-java repository. A better place would be the parquet site: https://parquet.apache.org/docs/overview/

Reply:
Got it , added some examples in Python and C++